computer_assisted_intervention/video_seg.ipynb

```diff
@@ -374,10 +374,11 @@
     "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy. \n",
     "There is a pretrained bundle for surgical tool segmentation task in the MONAI model-zoo, the model can be automatically downloaded and loaded to the network by adding codes to the block below:\n",
     "```\n",
-    "pretrained_weights = monai.bundle.load(\n",
-    "    name=\"endoscopic_tool_segmentation\", bundle_dir=\"./\", version=\"0.2.0\"\n",
-    ")\n",
-    "model.load_state_dict(pretrained_weights)\n",
+    "pretrained_weights = monai.bundle.load(name=\"endoscopic_tool_segmentation\", bundle_dir=\"./\", version=\"0.2.0\")\n",
+    "model_weight = model.state_dict()\n",
+    "weights_no_head = {k:v for k,v in pretrained_weights.items() if not \"segmentation_head\" in k}\n",
```

Comment:
Maybe you can use this API of MONAI to easily load the expected weights?
https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/utils.py#L449
CC @yiheng-wang-nv @wyli .

Thanks.